Ass_4/src/Account.java

```diff
@@ -1,40 +1,53 @@
 public class Account {
     private String name; //username;
-    private int value; // account balance;
+    private int balance; // account balance;
 
-    public void putMoney(int money){
-        this.value = this.value + money;
+
+    public Account(String name)
```

Comment:
Update the constructor to initialize the account balance as well instead of hard coding and then work with the account balance.